sycl/test-e2e/Matrix/SG32/get_coord_float_matC.cpp

```diff
@@ -10,7 +10,6 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: cpu
 
```

Comment:
These tests should have xpassed upon OCL CPU 2024.0 uplift to SYCLOS as the feature was added in 2024.0.
Adding @Nuullll. 
AFAIK, they started xpassing very recently.
It is worth investigating why it took time for the uplift to happen.
But AFAIK, the uplift to make these tests pass already happened.

Reply:
We implemented the feature in 2024.0 but there was another bug introduced by https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2214. So the test didn't pass with 2024.0.
The current uplift (2024.1) includes the fix for the bug, so these tests start passing as expected.